README.md

```diff
@@ -2,548 +2,282 @@
 
 # debug.rb
 
-This library provides debugging functionality to Ruby (MRI) 2.6 and later.
-
-This debug.rb is replacement of traditional lib/debug.rb standard library which is implemented by `set_trace_func`.
-New debug.rb has several advantages:
-
-* Fast: No performance penalty on non-stepping mode and non-breakpoints.
-* [Remote debugging](#remote-debugging): Support remote debugging natively.
-  * UNIX domain socket (UDS)
-  * TCP/IP
-  * Integration with rich debugger frontends
-
-     Frontend |  [Console](https://github.com/ruby/debug#invoke-as-a-remote-debuggee) | [VSCode](https://github.com/ruby/debug#vscode-integration) | [Chrome DevTool](#chrome-devtool-integration) |
+This library provides debugging functionality to Ruby (MRI) 2.6 and later. It has several advantages:
+
+- Fast: No performance penalty on non-stepping mode and non-breakpoints.
+- Native remote debugging support:
+  - [UNIX domain socket](/docs/remote_debugging.md#unix-domain-socket-uds) (UDS)
+  - [TCP/IP](/docs/remote_debugging.md#tcpip)
+  - Integration with rich debugger frontends
+     Frontend |  [Console](/docs/remote_debugging.md#debugger-console) | [VSCode](/docs/remote_debugging.md#vscode) | [Chrome DevTools](/docs/remote_debugging.md#chrome-devtool-integration) |
      ---|---|---|---|
      Connection | UDS, TCP/IP | UDS, TCP/IP | TCP/IP |
      Requirement | No | [vscode-rdbg](https://marketplace.visualstudio.com/items?itemName=KoichiSasada.vscode-rdbg) | Chrome |
-
-* Extensible: application can introduce debugging support with several ways:
-  * By `rdbg` command
-  * By loading libraries with `-r` command line option
-  * By calling Ruby's method explicitly
-* Misc
```

Comment:
The points here are not clear to users without further explanation, so I removed them for now.